From b45bbd9ac8fd01946740e373003833e58296c3dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ignacy=20Kuchci=C5=84ski?= <ignacykuchcinski@gmail.com>
Date: Tue, 17 Jun 2025 15:57:25 +0200
Subject: [PATCH 3/5] cups: Add avahi-libs dependency

Currently we build cups without avahi, which results in skipping
building of ippfind utility, which is needed for printer discovery.

To fix that, add avahi-libs as build dependency.
---
 elements/components/cups-base.bst | 1 +
 elements/components/cups.bst      | 1 +
 2 files changed, 2 insertions(+)

diff --git a/elements/components/cups-base.bst b/elements/components/cups-base.bst
index af3815280..93efebdfd 100644
--- a/elements/components/cups-base.bst
+++ b/elements/components/cups-base.bst
@@ -5,6 +5,7 @@ build-depends:
 - components/gzip.bst
 - components/openssl.bst
 - components/systemd.bst
+- components/avahi-libs.bst
 
 variables:
   # Makefiles not added to build-dir which makes it unusable
diff --git a/elements/components/cups.bst b/elements/components/cups.bst
index 1a13bb909..59214b221 100644
--- a/elements/components/cups.bst
+++ b/elements/components/cups.bst
@@ -4,6 +4,7 @@ build-depends:
 - components/cups-base.bst
 
 runtime-depends:
+- components/avahi-libs.bst
 - components/openssl.bst
 - components/cups-manifest.bst
 
-- 
2.49.0

